src/plugin/coreplugins/CustomPluginConfig/widget/ConfigWidget.js

```diff
@@ -0,0 +1,52 @@
+/*globals, define, $, WebGMEGlobal*/
```

Comment:
The API is so small and the base class wouldn't really provide any functionality.